docs/sphinx/examples/metadata-io.cpp

```diff
@@ -109,8 +111,30 @@ namespace
                << "\n\tY = " << meta.getPixelsSizeY(i)
                << "\n\tZ = " << meta.getPixelsSizeZ(i)
                << "\n\tT = " << meta.getPixelsSizeT(i)
-               << "\n\tC = " << meta.getPixelsSizeC(i)
-               << '\n';
+               << "\n\tC = " << meta.getPixelsSizeC(i);
+        // These are optional, so handle failure gracefully.
+        try
+          {
+            stream << "\n\tPhysicalX = " << meta.getPixelsPhysicalSizeX(i);
+          }
+        catch (const meta::MetadataException&)
+          {
+          }
+        try
+          {
+            stream << "\n\tPhysicalY = " << meta.getPixelsPhysicalSizeY(i);
+          }
+        catch (const meta::MetadataException&)
+          {
+          }
+        try
+          {
+            stream << "\n\tPhysicalZ = " << meta.getPixelsPhysicalSizeZ(i);
+          }
+        catch (const meta::MetadataException&)
+          {
+          }
+        stream<< '\n';
 
```

Comment:
That would mean skipping potentially valid properties--any of these could be present or absent, so could mean missing out on following properties.